app/views/content/events/get-the-most-from-events/_content.html.erb

```diff
@@ -110,6 +110,19 @@
       <h3>Event staff</h3>
       <p> Our friendly team will be there to welcome you at the door. Ask them about the event, where to find the experts, and where you can get overseas qualifications checked.</p>
 
+    <h3>What a Get Into Teaching Event will be like</h3>
+
+<p>Watch our video to see what happens at a Get Into Teaching event and how it could help you make a decision.</p>
```

Comment:
```suggestion
<p>Watch our video to see what happens at a Get Into Teaching event and how it could help you make a decision about teaching.</p>
```
Is this a decision about attending the event or about becoming a teacher - presumably the latter? Is it worth making this clear (reword as needed!)?